.github/workflows/build.yml

```diff
@@ -39,25 +42,54 @@ jobs:
           context: .
           file: ./CI/Dockerfile
           platforms: linux/amd64
-          tags: lib3mf_ubi8:latest       
-          load: true   
-      - 
-        name: Docker Extract                    
+          tags: lib3mf_ubi8:latest
+          load: true
+      -
+        name: Docker Extract
         uses: shrink/actions-docker-extract@v3.0.0
         id: extract
-        with: 
+        with:
           image: lib3mf_ubi8:latest
           path: out.zip
           destination: dist
-          
+
       - run: unzip out.zip
         working-directory: ./dist
-      -
-        name: Upload Artifact
+      - name: Upload Artifact
         uses: actions/upload-artifact@v2
         with:
           name: lib3mf.so
-          path: dist/lib3mf.so.2   
+          path: dist/lib3mf.so.2
+      - name: Extract File Name (CPacked Archive)
+        run: |
+          ZIP_FILE=$(ls dist/lib3mf-*.zip)
+          echo "ARTIFACT_NAME_ZIP=$(basename ${ZIP_FILE})" >> $GITHUB_ENV
```

Comment:
Are these echo useful? When I check the logs its showing as the same: 

ZIP_FILE=$(ls dist/lib3mf-*.zip)
  echo "ARTIFACT_NAME_ZIP=$(basename ${ZIP_FILE})" >> $GITHUB_ENV
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    LIB[3](https://github.com/3MFConsortium/lib3mf/actions/runs/9339870424/job/25704782439#step:13:3)MF_VERSION: 2.3.1
    RUN_INTEGRATION_TESTS: on

Reply:
ignore this comment. I misunderstood that echo would print the artifact name in log.